src/main/java/gov/usds/case_issues/controllers/CaseDetailsApiController.java

```diff
@@ -69,11 +77,21 @@ public CaseDetails getCaseDetails(@PathVariable String caseManagementSystemTag,
 		return ResponseEntity.ok(replacement);
 	}
 
-	@PostMapping("activeSnooze/notes")
+	@GetMapping("attachments")
+	public ResponseEntity<List<AttachmentSummary>> getAttachments(@PathVariable String caseManagementSystemTag,
+			@PathVariable String receiptNumber) {
+		TroubleCase rootCase = _caseDetailsService.findCaseByTags(caseManagementSystemTag, receiptNumber);
+		List<AttachmentSummary> attachments = _attachmentService.findAttachmentsForCase(rootCase).stream()
+				.map(AttachmentSummary::new)
+				.collect(Collectors.toList());
+		return ResponseEntity.ok(attachments);
+	}
+
+	@PostMapping({"activeSnooze/notes","activeSnooze/attachments"})
 	@RequireUpdateCasePermission
-	public ResponseEntity<?> addNote(@PathVariable String caseManagementSystemTag,
-			@PathVariable String receiptNumber, @RequestBody AttachmentRequest newNote) {
-		_caseDetailsService.annotateActiveSnooze(caseManagementSystemTag, receiptNumber, newNote);
-		return ResponseEntity.accepted().build();
+	public ResponseEntity<AttachmentSummary> addAttachment(@PathVariable String caseManagementSystemTag,
+			@PathVariable String receiptNumber, @RequestBody AttachmentRequest attachment) {
+		CaseAttachmentAssociation attached = _caseDetailsService.annotateActiveSnooze(caseManagementSystemTag, receiptNumber, attachment);
+		return ResponseEntity.created(null).body(new AttachmentSummary(attached));
```

Comment:
We should have a GET/DELETE/PATCH endpoint for individual note associations, but for now we don't and I don't think putting in a Location header that tells lies about that is wise.